### PR TITLE
Add registration page

### PR DESCRIPTION
**Pull request by Nghondzweni**
I set the pull request to draft, there are still a bit of changes that I want to make. But overall the page is relatively complete.

### CODE REVIEW

**Review by longtomjr — CHANGES_REQUESTED** [review]
There seems to be a massive package-lock change. It might be a npm version issue on your end. We will just need to resolve that before merging.

File: web-client/src/app/views/register/register.page.ts [hunk 1]
```diff
@@ -1,31 +1,68 @@
...
-    })
-  };
-
+      firstName: ['', Validators.compose([Validators.required])],
+      lastName: ['', Validators.compose([Validators.required])],
+      mobile: [
+        '',
+        Validators.compose([
+          Validators.required,
+          Validators.minLength(1),
```
Comment (jongbonga) [inline web-client/src/app/views/register/register.page.ts:27]: I think minLength here should be 10

Comment (longtomjr) [reply web-client/src/app/views/register/register.page.ts:27]: Min length is 4 and max is 10 from what we discussed.

**Comment by longtomjr** [comment]
Did a rebase to revert the package lock change.

**Review by longtomjr — APPROVED** [review]

**Review by jongbonga — APPROVED** [review]

**Pull request merged**